### PR TITLE
Add info on "Using Apache Kafka and OpenSearch to explore Mastodon" session

### PR DESCRIPTION
**Pull request by anelook**
### Description
Add information about session "Using Apache Kafka and OpenSearch to explore Mastodon" at DevoxxUK in London
 

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _events/2023-0510-devoxxuk.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tz: UTC
+title: Using Apache Kafka and OpenSearch to explore Mastodon
+online: false
+signup:
+    url: https://www.devoxx.co.uk/
+    title: Join Devoxx UK 2023 conference in London
+
+---
+
+Join Olena Kutsenko at DevoxxUK for the session "Using Apache Kafka and OpenSearch to explore Mastodon"
```
Comment (krisfreedain) [inline _events/2023-0510-devoxxuk.markdown:13]: Would suggest adding your company/title after your name. Also, consider linking to your Twitter/GitHub/or event profile so the audience knows 😃

looks like this is a link for your profile/talk - maybe add that in the post somewhere as well: https://www.devoxx.co.uk/speaker-details/?id=7861

**Review by krisfreedain — CHANGES_REQUESTED** [review]
one small suggestion

**Comment by krisfreedain** [comment]
Hello @anelook - DCO is unfortunately a requirement for the repo - are you able to push through with that? 

also, left a small edit suggestion. thanks!

File: _events/2023-0510-devoxxuk.markdown (hunk 1 above)
Comment (anelook) [reply _events/2023-0510-devoxxuk.markdown:13]: Good idea! Added this

**Comment by anelook** [comment]
Thank you, @krisfreedain, added the link and fixed DCO :)

**Review by krisfreedain — APPROVED** [review]
thanks @anelook!

**Pull request merged**